plugins/hypervisors/kvm/src/main/java/com/cloud/ha/KVMInvestigator.java

```diff
@@ -101,24 +115,29 @@ public Status isAgentAlive(Host agent) {
                 hostStatus = answer.getResult() ? Status.Down : Status.Up;
             }
         } catch (Exception e) {
-            s_logger.debug("Failed to send command to host: " + agent.getId());
+            s_logger.debug(String.format("Failed to send command to %s", agent));
```

Comment:
- Should an exception be logged in `debug`? 
- We could pass the exception as parameter to the log.
- Do we need a Pokemon here?

Reply:
Thanks for bringing it up @GutoVeronezi, these are pertinent questions.

This is a perfect case of _Pokémon catch_ (catch them all). To be honest, I cannot even find how an Exception could be raised by `AgentManagementImpl.easySend` as the method itself handles `Exception`.

I was inclined to simply remove it; however, I avoided changing any behavior of the HeartBeat execution flow avoiding any regression issue.

I will take a double-check at this `easySend` catch.
But it is a good thing to pay attention to, if I do not change it at this PR we can always plan another PR/issue cleaning some of these catches.

Reply:
@GutoVeronezi I decided to remove this catch.
When checking the easySend there is already enough catches. If it does not catch the exception ... I don't know what would catch it:

```
public Answer easySend(final Long hostId, final Command cmd) {
        try {
                ...
                ...
                ...
        } catch (final AgentUnavailableException e) {
            s_logger.warn(e.getMessage());
            return null;
        } catch (final OperationTimedoutException e) {
            s_logger.warn("Operation timed out: " + e.getMessage());
            return null;
        } catch (final Exception e) {
            s_logger.warn("Exception while sending", e);
            return null;
        }
```

Reply:
@GabrielBrascher indeed, I see no way to it throw an exception here